src/xrGame/script_game_object_inventory_owner.cpp

```diff
@@ -1147,7 +1146,6 @@ void CScriptGameObject::HideWeapon()
 {
 #ifdef DEBUG
     GEnv.ScriptEngine->script_log(LuaMessageType::Message, "CScriptGameObject::HideWeapon called!!!");
-    GEnv.ScriptEngine->print_stack();
```

Comment:
This change is not relevant to this PR, please, revert the change. I'll remove this debug code in separate commit :)